src/hera.cpp

```diff
@@ -67,34 +68,39 @@ const map<string, WasmEngineCreateFn> wasm_engine_map {
 #endif
 };
 
-const map<string, hera_evm1mode> evm1mode_options {
-  { "reject", hera_evm1mode::reject },
-  { "fallback", hera_evm1mode::fallback },
-  { "evm2wasm", hera_evm1mode::evm2wasm_contract },
-};
-
-struct hera_instance : evmc_instance {
-  unique_ptr<WasmEngine> engine{
+WasmEngineCreateFn wasmEngineCreateFn =
 // This is the order of preference.
 #if HERA_BINARYEN
-    new BinaryenEngine
+    BinaryenEngine::create
```

Comment:
@chfast hmm, why did we had the static `create()` if we only ever used the default constructor?

Reply:
I don't remember anything about it. If the default constructor works for this case it's good choice.